### ISSUE TITLE
ScopeManagerShim.activate(span) throws IllegalArgumentException when null is passed in

### ISSUE THREAD

**Issue by shawnsong** [open-telemetry/opentelemetry-java#3952]
`ScopeManagerShim.activate()` throws `IllegalArgumentException` if a `null` span is passed in. I'm not sure whether this is a bug or implemented on purpose but it is inconvenient to use as we need to do a `null` check before calling `activate`.

Currently, I need to do this to avoid the exception.
```
try (Scope scope = (span != null ? tracer.scopeManager().activate(span) : null)) {
  ...
}
```
It would be good if `null` is returned by the method as try-with-resources doesn't complain a `null` resource. 

**What version and what artifacts are you using?**
Artifacts: (`opentelemetry-opentracing-shim`)
Version: (`v1,9,1-alpha`)

**Comment by carlosalberto** [comment]
Fixed via #3985

**Comment by anpatellll** [comment]
@carlosalberto Whats the ETA of this being released?
